bigtable/tests/system.py

```diff
@@ -356,6 +356,32 @@ def _write_to_row(self, row1=None, row2=None, row3=None, row4=None):
         cell4 = Cell(CELL_VAL4, timestamp4)
         return cell1, cell2, cell3, cell4
 
+    def test_mutate_rows(self):
+        row_1 = self._table.row(ROW_KEY)
+        row_1.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL1)
+        row_1.commit()
+        row_2 = self._table.row(ROW_KEY_ALT)
+        row_2.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL2)
+        row_2.commit()
+        rows = [row_1, row_2]
+        self.rows_to_delete.extend(rows)
+
+        # Change the contents
+        row_1.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL3)
+        row_2.set_cell(COLUMN_FAMILY_ID1, COL_NAME1, CELL_VAL4)
+        statuses = self._table.mutate_rows(rows)
+        result = [status.code for status in statuses]
+        expected_result = [0, 0]
+        self.assertEqual(result, expected_result)
+
+        # Check the contents
+        row_1_data = self._table.read_row(ROW_KEY)
+        self.assertEqual(
+            row_1_data.cells[COLUMN_FAMILY_ID1][COL_NAME1][0].value, CELL_VAL3)
```

Comment:
I would prefer to keep the current version. 
When I construct cells, they are the different instances of Cell class, because the system test return true value from test BigTable server.
I can either mock the response from BT or compare exact values of Cell instances.
In the first case the test will became not system anymore. It the second: I won't avoid `getattr`/`__getitem__`.